src/bin/vip.js

```diff
@@ -107,10 +107,13 @@ const rootCmd = async function() {
 		await trackEvent( 'login_command_token_submit_success' );
 
 		// Exec the command we originally  wanted
-		const spawn = require( 'child_process' ).spawn;
-		const bin = process.argv[ 1 ];
 		const argv = process.argv.slice( 2 );
-		spawn( bin, argv, { stdio: 'inherit' } );
+		if ( argv.length ) {
+			console.log( argv );
+			return args.runCommand( { usage: process.argv.slice( 2 ) } );
+		}
+
+		return args.showHelp();
```

Comment:
Because we only define these when a token is set, we end up with an empty help:

```
  Usage: vip.js [options] [command]

  Commands:

    help  Display help

  Options:

    -h, --help     Output usage information
    -v, --version  Output the version number
```